src/SmartComponents/Inventory/applications/advisor/Advisor.js

```diff
@@ -66,7 +68,7 @@ class InventoryRuleList extends Component {
 
     async fetchKbaDetails (kbaIds) {
         try {
-            const data = await fetch(`/rs/search?q=id:(${kbaIds})&fl=view_uri,id,publishedTitle`, { credentials: 'include' })
+            const data = await fetch(`https://access.redhat.com/rs/search?q=id:(${kbaIds})&fl=view_uri,id,publishedTitle`, { credentials: 'include' })
```

Comment:
Do we realy want to include `https://access.redhat.com` in here? I guess the best way would be to add this to spandx locally and to 3scale as well so once this is migrated to c.rh.com we are good to go with removing these rules from these and not by changing code.

Reply:
If you gotta better way to do this, yes, please I am all ears, would you mind dropping in a commit @karelhala ?

Reply:
My bad, I didn't understood the reason why you want to include it there. Good to go!